apps/web/app/globals.css

```
@@ -777,18 +777,31 @@
 
 .markdown-body th,
 .markdown-body td {
   border: 1px solid var(--border);
   padding: 0.25rem 0.5rem;
   font-size: 13px;
+  vertical-align: top;
+}
+
+.markdown-body td {
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)